scripts/kind.sh

```diff
@@ -14,14 +14,40 @@ source "$SCRIPTS_DIR/lib/logging.sh"
 setup_kind_cluster() {
     local __cluster_name=$1
     local __controller_namespace=$2
-    local kubeconfig_path="$ROOT_DIR/build/clusters/$__cluster_name/kubeconfig"
+    local kind_cluster_build_dir="$ROOT_DIR/build/clusters/$__cluster_name"
+    local kubeconfig_path="$kind_cluster_build_dir/kubeconfig"
+    local kubecontext="kind-$__cluster_name"
```

Comment:
nit: Do we want to add an `ack` specific prefix to the name of the context to differentiate it between the other kind clusters?

Reply:
I suppose that would be a good idea, yeah. I'll add.

Reply:
@RedbackThomson actually, I just realized, the `$__cluster_name` already has `ack` in it (ex: `ack-test-a485abb9`) so I think we're good here.

Reply:
Oh awesome. Great